Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1957,6 +1959,17 @@ def get_logs_defender_download_command(client: PrismaCloudComputeClient, args: d
     return fileResult(f"{hostname}-logs.tar.gz", response, entryTypes["entryInfoFile"])
 
 
+def unstuck_fetch_stream_command():
+    ctx = demisto.getIntegrationContext()
+    demisto.debug(f"Integration Context before update = {ctx}")
```

Comment:
What happens once the integration context is huge? Why not print to logs only the value?